packages/react/runtime/jsx-dev-runtime/index.d.ts

```diff
@@ -3,3 +3,16 @@
 // LICENSE file in the root directory of this source tree.
 export { jsxDEV, Fragment } from 'react/jsx-dev-runtime';
 export { jsx, jsxs } from 'react/jsx-runtime';
+import { IntrinsicElements as _IntrinsicElements } from '@lynx-js/types';
+import { JSX as _JSX } from 'react';
+
+export namespace JSX {
+  interface IntrinsicElements extends _IntrinsicElements {
+  }
+
+  interface IntrinsicAttributes {
+    ref?: unknown | ((e: unknown) => void) | undefined;
```

Comment:
The `unknown` does not look correct. You may keep the type same as before https://github.com/lynx-family/lynx-stack/blob/632bdddcf065262b98f06cf451a4d0d129e4a46a/packages/react/types/react.d.ts#L97

```suggestion
    ref?: Ref<NodesRef>;
```

Reply:
Actually, I don't think we need `ref` here since we already have `ref` in the `StandardProps`, and it will be the base interface of `ViewProps` https://github.com/lynx-family/lynx/blob/8198dcbaaf8f0a3334eb730664fa15e6f138b8d9/js_libraries/types/types/common/element/view.d.ts#L7

```js
interface ViewProps extends StandardProps {}
```

Reply:
Yes, I found the type of ref here:
https://github.com/lynx-family/lynx-stack/blob/47e5b22715a39b5fbd6c4c4cc734293a3ccd75b8/packages/react/types/react.d.ts#L97
I'll delete.